jobs/payment-jobs/tests/jobs/test_cfs_create_account_task.py

```diff
@@ -124,5 +124,5 @@ def test_update_pad_account(session):
     assert updated_cfs_account.bank_number == new_payment_details.get('bankInstitutionNumber')
 
     assert cfs_account.status == CfsAccountStatus.INACTIVE.value
-    assert updated_cfs_account.status == CfsAccountStatus.PENDING_PAD_ACTIVATION.value
+    assert updated_cfs_account.status == CfsAccountStatus.ACTIVE.value
```

Comment:
this test case has to be rechecked...

Reply:
Yes. I will run in local to check this.

Reply:
This is correct. We moved the logic to update the bank information right away if PAD information is getting updated, which will result in ACTIVE cfs_account.